packages/editor-ui/src/composables/useWorkflowHelpers.ts

```diff
@@ -349,39 +348,6 @@ function connectionInputData(
 		}
 	}
 
-	const workflowsStore = useWorkflowsStore();
-
-	if (workflowsStore.shouldReplaceInputDataWithPinData) {
```

Comment:
I see another use of `shouldReplaceInputDataWithPinData` in `useWorkflowHelpers` - that we cannot simplify also?

Reply:
I checked that. That was actually necessary to set the input data for `$input` and `$json` to resolve for example.